package.json

```diff
@@ -57,7 +57,7 @@
     "ember-moment": "4.2.1",
     "ember-notify": "4.3.1",
     "ember-responsive": "1.2.3",
-    "ivy-codemirror": "~1.2.0",
+    "ivy-codemirror": "piotrpalek/ivy-codemirror#pass-params",
```

Comment:
It's ok if we can't, but I'd much prefer to make sure the upstream changes land before merging.